src/components/shop/item-registry.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * @copyright BBC 2020
+ * @author BBC Children's D+E
+ * @license Apache-2.0
+ */
+
+import fp from "../../../lib/lodash/fp/fp.js";
+
+export const initRegistry = itemsArray => {
+    const items = fp.clone(itemsArray);
```

Comment:
it doesn't look like we're mutating itemsArray anywhere, so we _might_ be ok just referencing that rather than shallow copying it

Reply:
ab34489